backend/hdf5/h5x/H5DataType.cpp

```diff
@@ -76,6 +77,15 @@ H5T_sign_t DataType::sign() const {
     return res;
 }
 
+void DataType:: cset(H5T_cset_t cset) {
```

Comment:
stray blank here? `DataType:: cset()`
no blank lines between setter and getter?